test/runScript.spec.js

```diff
@@ -146,7 +147,7 @@ describe('runScript', () => {
       await taskPromise
     } catch (err) {
       expect(err.message)
-        .toMatch(`🚫 mock-fail-linter found some errors. Please fix them and try committing again.
+        .toMatch(`${logSymbols.error} mock-fail-linter found some errors. Please fix them and try committing again.
```

Comment:
Same here: Should we switch to http://facebook.github.io/jest/docs/en/expect.html#tothrowerrormatchingsnapshot instead